unpackaged/config/qa/objects/Account.object

```diff
@@ -12,30 +12,6 @@
         <trackTrending>false</trackTrending>
         <type>Currency</type>
     </fields>
-    <fields>
```

Comment:
These two fields used to be used in the levels robot tests, and more specifically I use them in the levels tests I expanded on -- It would validate that the "Current Level" field was populated after running the batch job.  For those tests to work, there would need to be a Level__c lookup field on the Account (or Contact) object. I can see where we might not want to have those fields in all orgs, though the `A_User_Defined_Rollup_Field__c` field is still being created in this `.object` file. 

I'm unclear on the usage of the different folders. If `unpackaged/config/qa` is for QA testing, then I'd expect these fields to be needed. But, maybe this means only manual QA testing?

Reply:
The trial org includes these level fields for Account (although it calls the current one Level__c instead of Current_Level__c) so I moved them to unpackaged/config/trial. I'm thinking of unpackaged/config/trial as the standard NPSP configuration for new non-dev orgs. QA orgs get that PLUS a small amount of additional QA-only configuration in unpackaged/config/qa (basically some fields for testing that don't belong in customer orgs).